### PR TITLE
Cleaning up workspace clone information messages

### PR DESCRIPTION
**Pull request by sunix**
https://github.com/eclipse/che/issues/11838

Signed-off-by: Sun Tan <sutan@redhat.com>

### CODE REVIEW

File: src/che-workspace-project-manager.ts [hunk 1]
```diff
@@ -32,6 +32,9 @@ export class CheWorkspaceProjectManager {
         }
 
         const cloneCommandList = await this.selectProjectToCloneCommands(workspaceConfig);
+        if (cloneCommandList.length <= 0) {
+            return;
```
Comment (benoitf) [inline src/che-workspace-project-manager.ts:36]: On which case can it be < 0 ? Why not === 0

Comment (sunix) [reply src/che-workspace-project-manager.ts:36]: we don't know ...

Comment (benoitf) [reply src/che-workspace-project-manager.ts:36]: ? the function is implemented in the current class, it can never return a negative value

Comment (sunix) [reply src/che-workspace-project-manager.ts:36]: Technically array.length is a number and can be a negative value. it depends on the implementation of the JS runtime not ? but i don't mind :)
I have update.

Comment (benoitf) [reply src/che-workspace-project-manager.ts:36]: > Every Array object has a length property whose value is always a nonnegative integer less than 232.
http://www.ecma-international.org/ecma-262/5.1/index.html#sec-15.4

Comment (sunix) [reply src/che-workspace-project-manager.ts:36]: yes this is the spec

**Review by benoitf — APPROVED** [review]
forgot to approve

**Pull request merged**